client/platform/desktop/frontend/components/ViewerLoader.vue

```diff
@@ -51,41 +51,48 @@ export default defineComponent({
     const compoundId = ref(props.id);
     const subTypeList = computed(() => [datasets.value[props.id]?.subType || null]);
     const camNumbers = computed(() => [datasets.value[props.id]?.cameraNumber || 1]);
-    const readOnlyMode = computed(() => settings.value?.readonlyMode || false);
+    const readonlyMode = computed(() => settings.value?.readonlyMode || false);
+    const selectedCamera = ref('');
```

Comment:
`ViewerLoader` is outside of `Viewer`.  Cameras only get selected and changed if there are multiple cameras.  This means the `datasetId` remains standard for the case in which the camera isn't changed.  The `modifiedId` which is used for running pipelines is dependent on the `selectedCamera` truthiness.  If it remains `''` then it uses the base Id.  This allows standard cameras to operate normally for pipelines and then when you have multi-camera datasets it will use the selectec camera's Id:  `{datasetId}/{cameraName}`.  There is probably a better way to convey this in the code than how I did it.